src/features/workflow/CodeEditor.tsx

```diff
@@ -81,6 +81,19 @@ const CodeEditor: React.FC<CodeEditorProps> = ({
     }
   }, [theEditor])
 
+  useEffect(() => {
+    theEditor?.addAction({
+      id: "executeCurrentAndAdvance",
+      label: "Execute Block and Advance",
+      keybindings: [KeyMod.CtrlCmd | KeyCode.Enter],
+      contextMenuGroupId: "2_execution",
+      precondition: "editorTextFocus && !suggestWidgetVisible && !renameInputVisible && !inSnippetMode && !quickFixWidgetVisible",
+      run: () => {
+        onRun()
+      }
+    })
+  }, [theEditor, onRun])
+
```

Comment:
this is new functionality, correct?

If so, add a comment in the main PR comment saying that you are adding the ability to dry run a workflow by doing "cmd+enter" or "ctrl-enter" as part of this PR.

Reply:
Its not a new functionality. It was there, but problem was that it was doing it on component mount. So latestDryRunId was neglected there. That is why I have added this code. @ramfox

Reply:
ahh. Okay, so because `onRun` now changes when the `latestDryRunId` changes, `handleEditorDidMount` is not properly adding the `onRun` action because it only happens on mount.

Do we still need `handleEditorDidMount`? If so, can you add a comment to it explaining why we need both. I can see it being confusing in the future to see the same thing happening twice.

Also, why don't we need to call `setTheEditor` here?

Reply:
@ramfox , did slight modifications here. We do still need the handleEditorDidMount as this is the only place we are initializing `theEditor` variable. If I call the `setTheEditor` here it breaks the UI changes by previous useEffect that determine the height of the cell. I think it is because we are not setting theEditor at any point of the component and working with the object itself. As I didn't really worked with monaco editor before I just thought that this is the way to work with it: just initialize it and then use internal method to make changes. @ramfox